### PR TITLE
refactor: Move constants outside of variable scope

### PR DESCRIPTION
**Pull request by CaedenPH**
### Describe your change:
Moves constants outside of variable scope, removing the need for `noqa: N806`

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x]  All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: ciphers/brute_force_caesar_cipher.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```
Comment (cclauss) [inline ciphers/brute_force_caesar_cipher.py:1]: Please import this from the `strings` module.

Comment (CaedenPH) [reply ciphers/brute_force_caesar_cipher.py:1]: Sure, I just moved the constants outside the function scope

File: conversions/decimal_to_any.py [hunk 2]
```diff
@@ -1,6 +1,15 @@
 """Convert a positive Decimal Number to Any Other Representation"""
 
 
+# fmt: off
+ALPHABET_VALUES = {'10': 'A', '11': 'B', '12': 'C', '13': 'D', '14': 'E', '15': 'F',  # noqa: E501
+                   '16': 'G', '17': 'H', '18': 'I', '19': 'J', '20': 'K', '21': 'L',
+                   '22': 'M', '23': 'N', '24': 'O', '25': 'P', '26': 'Q', '27': 'R',
+                   '28': 'S', '29': 'T', '30': 'U', '31': 'V', '32': 'W', '33': 'X',
+                   '34': 'Y', '35': 'Z'}
+# fmt: on
```
Comment (dhruvmanila) [inline conversions/decimal_to_any.py:10]:
```suggestion
ALPHABET_VALUES = {'10': 'A', '11': 'B', '12': 'C', '13': 'D', '14': 'E', '15': 'F',
                   '16': 'G', '17': 'H', '18': 'I', '19': 'J', '20': 'K', '21': 'L',
                   '22': 'M', '23': 'N', '24': 'O', '25': 'P', '26': 'Q', '27': 'R',
                   '28': 'S', '29': 'T', '30': 'U', '31': 'V', '32': 'W', '33': 'X',
                   '34': 'Y', '35': 'Z'}
```

File: web_programming/fetch_quotes.py [hunk 3]
```diff
@@ -12,13 +12,13 @@
 
 
 def quote_of_the_day() -> list:
-    API_ENDPOINT_URL = "https://zenquotes.io/api/today/"  # noqa: N806
-    return requests.get(API_ENDPOINT_URL).json()
+    api_endpoint_url = "https://zenquotes.io/api/today/"
+    return requests.get(api_endpoint_url).json()
```
Comment (dhruvmanila) [inline web_programming/fetch_quotes.py:16]: This (`API_ENDPOINT_URL`) is a constant, we can keep it outside.

**Review by dhruvmanila — CHANGES_REQUESTED** [review]

**Comment by cclauss** [comment]
Did you use some tool to find these instances?

File: compression/peak_signal_to_noise_ratio.py [hunk 4]
```diff
@@ -11,14 +11,16 @@
...
 def psnr(original: float, contrast: float) -> float:
     mse = np.mean((original - contrast) ** 2)
     if mse == 0:
         return 100
-    PIXEL_MAX = 255.0  # noqa: N806
-    PSNR = 20 * math.log10(PIXEL_MAX / math.sqrt(mse))  # noqa: N806
-    return PSNR
+
+    psnr = 20 * math.log10(PIXEL_MAX / math.sqrt(mse))
+    return psnr
```
Comment (cclauss) [inline compression/peak_signal_to_noise_ratio.py:23]: `flake8-return` would have suggested this and given that `psnr` is not at all self-documenting, I agree.
```suggestion
    return 20 * math.log10(PIXEL_MAX / math.sqrt(mse))
```

File: compression/peak_signal_to_noise_ratio.py [hunk 5]
```diff
@@ -11,14 +11,16 @@
 import cv2
 import numpy as np
 
+PIXEL_MAX = 255.0
+
 
 def psnr(original: float, contrast: float) -> float:
```
Comment (cclauss) [inline compression/peak_signal_to_noise_ratio.py:17]:
```suggestion
def peak_signal_to_noise_ratio(original: float, contrast: float) -> float:
```

File: conversions/decimal_to_any.py [hunk 6]
```diff
@@ -1,6 +1,36 @@
...
+    "27": "R",
+    "28": "S",
+    "29": "T",
+    "30": "U",
+    "31": "V",
+    "32": "W",
+    "33": "X",
+    "34": "Y",
+    "35": "Z",
+}
```
Comment (cclauss) [inline conversions/decimal_to_any.py:31]: from string import ascii_uppercase

ALPHABET_VALUES = {str(ord(c) - 55): c for c in ascii_uppercase}

Comment (cclauss) [inline conversions/decimal_to_any.py:31]:
```suggestion
from string import ascii_uppercase

ALPHABET_VALUES = {str(ord(c) - 55): c for c in ascii_uppercase}
```

File: conversions/decimal_to_any.py [hunk 7]
```diff
@@ -65,13 +95,6 @@ def decimal_to_any(num: int, base: int) -> str:
         raise ValueError("base must be >= 2")
     if base > 36:
         raise ValueError("base must be <= 36")
-    # fmt: off
```
Comment (cclauss) [inline conversions/decimal_to_any.py:68]: In general, we should not remove `# fmt: off` and `# fmt: on` directives for psf/black.
It should continue to be the author's prerogative to compress uninteresting details.

Comment (CaedenPH) [reply conversions/decimal_to_any.py:68]: Yep, I fully intended to move the fmt's, I just must've mistyped

Comment (CaedenPH) [reply conversions/decimal_to_any.py:68]: Oh, @cclauss, it was dhruvmanila that removed the fmt's in a commit

Comment (cclauss) [reply conversions/decimal_to_any.py:68]: I was not trying to point a finger.  I was just making a suggestion.  The directive (like # noqa and # fmt) are there for a reason.

**Comment by cclauss** [comment]
If I have closed any failing PRs that you want to keep open, please reopen them or ask me to do so.

> Hacktoberfest is strenuous for our team of active volunteer maintainers. That small team now has ___200+ open pull requests___ to review.  We have __high standards for our submissions__ which are described in [`CONTRIBUTING.md`](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md#contributing-guidelines).  Our continuous integration tests help to ensure that our standards are met before we review and merge deserving pull requests.  This is further complicated by the fact that we already have [lots of algorithms](https://github.com/TheAlgorithms/Python/blob/master/DIRECTORY.md) and would like to avoid duplicates.

> For these reasons, I will **close all open pull requests failing tests** so that our unpaid team has a smaller mountain to climb.  If you want us to land your pull request, please make sure it is not a duplicate and then read [`CONTRIBUTING.md`](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md#contributing-guidelines) _carefully_ and try again.

**Review by dhruvmanila — APPROVED** [review]

**Comment by cclauss** [comment]
Your thoughts on using https://docs.python.org/3/library/typing.html#typing.Final for constant values?

**Comment by CaedenPH** [comment]
> Your thoughts on using https://docs.python.org/3/library/typing.html#typing.Final for constant values?

I think enforcing this will be quite difficult for not much benefit. What is your opinion on #7337?

**Pull request merged**